src/Emarsys/Client.php

```diff
@@ -1069,6 +1102,40 @@ protected function send($method = 'GET', $uri = '', array $body = [])
         return new Response($responseArray, (strpos($uri, '/rds') !== false && $method === 'GET'));
     }
 
+    /**
+     * send a request for given method, uri and body
+     *
+     * @param string $usedCSVDelimiter the delimiter for the csv data
+     * @param string $method           HTTP method to use
+     * @param string $uri              path to use with basePath for sending request, defaults to basePath
+     *
+     * @return array
+     * @throws \Emarsys\Exception\ClientException
+     * @throws \Emarsys\Exception\ServerException
+     */
+    protected function sendWithCSVResponse($usedCSVDelimiter, $method = 'GET', $uri = '')
+    {
+        $headers = [
```

Comment:
use `$this->send(...)` and handle the response as needed

Reply:
The current send method expecting a json response, where this "Emarsys reponse model" is built from.
But this endpoint will return a string with the export data.
The current "send" method will not work here.
So i created a new one.

But i will change the method name to a more explicit. That it is more obvious, what it should be used for.

Reply:
Then please change $responseJson to an other name otherwise a json should be expected